python_modules/libraries/dagster-dg/dagster_dg_tests/cli_tests/plus_tests/test_plus_login_command.py

```diff
@@ -1,28 +1,50 @@
 import json
-import os
 import queue
 import tempfile
 import threading
 import time
+from pathlib import Path
 from unittest import mock
 
 import pytest
 import requests
+import tomlkit
+import yaml
 from click.testing import CliRunner
 from dagster_dg.cli.plus import plus_group
-from dagster_shared.plus.config import DagsterPlusCliConfig, get_active_config_path
+from dagster_shared.plus.config import DagsterPlusCliConfig
 
 
 @pytest.fixture()
-def test_config_file(monkeypatch):
-    with tempfile.TemporaryDirectory() as tmpdir:
-        config_path = os.path.join(tmpdir, "config")
+def test_cloud_cli_config_file(monkeypatch):
```

Comment:
can we rename these fixtures not to start with `test_`? That can confuse some tooling which expects them to be actual test funcs